spec/system/authentication/user_logs_in_with_github_spec.rb

```diff
@@ -82,6 +121,35 @@
         )
       end
     end
+
+    context "when a validation failure occurrs" do
+      before do
+        OmniAuth.config.mock_auth[:github].extra.raw_info.name = "X" * 101
```

Comment:
What is this doing? Specifically the `"X" * 101`?

Reply:
Good point, let me add a comment about it. We have a name validator that renders names invalid if they are over 100 chars, so i created a string 101 chars long